integration-test/src/main/java/com/sequenceiq/it/cloudbreak/filesystem/FilesystemUtil.java

```diff
@@ -0,0 +1,129 @@
+                    + cloudProviderParams.get("accountKeyWasb");
+            CloudStorageAccount storageAccount = CloudStorageAccount.parse(storageConnectionString);
+            CloudBlobClient blobClient = storageAccount.createCloudBlobClient();
+            CloudBlobContainer container = blobClient.getContainerReference(containerName);
+            container.createIfNotExists();
+        } catch (Exception e) {
+            LOGGER.error("Error during creating the Wasb container", e);
+        }
+    }
+
+    protected static void deleteWasbContainer(Map<String, String> cloudProviderParams, String accountName, String containerName) {
+        try {
+            String storageConnectionString = "DefaultEndpointsProtocol=http;" + "AccountName=" + accountName + ";AccountKey="
+                    + cloudProviderParams.get("accountKeyWasb");
+            CloudStorageAccount storageAccount = CloudStorageAccount.parse(storageConnectionString);
+            CloudBlobClient blobClient = storageAccount.createCloudBlobClient();
+            CloudBlobContainer container = blobClient.getContainerReference(containerName);
+            container.deleteIfExists();
+        } catch (Exception e) {
+            LOGGER.error("Error during deleting the Wasb container", e);
+        }
+    }
+
+    protected static void deleteGcsObjects(ApplicationContext applicationContext, Map<String, String> cloudProviderParams, String bucketName,
+            String folderPrefix) throws Exception {
+        String serviceAccountPrivateKey = ResourceUtil.readBase64EncodedContentFromResource(applicationContext, cloudProviderParams.get("p12File"));
+        HttpTransport httpTransport = GoogleNetHttpTransport.newTrustedTransport();
+        PrivateKey privateKey = SecurityUtils.loadPrivateKeyFromKeyStore(SecurityUtils.getPkcs12KeyStore(),
+                new ByteArrayInputStream(Base64.decodeBase64(serviceAccountPrivateKey)), "notasecret", "privatekey", "notasecret");
+        JacksonFactory jsonFactory = JacksonFactory.getDefaultInstance();
+
+        GoogleCredential googleCredential = new GoogleCredential.Builder().setTransport(httpTransport)
+                .setJsonFactory(jsonFactory)
+                .setServiceAccountId(cloudProviderParams.get("serviceAccountId"))
+                .setServiceAccountScopes(Collections.singletonList(STORAGE_SCOPE))
+                .setServiceAccountPrivateKey(privateKey)
+                .build();
+
+        Storage storage = new Storage.Builder(httpTransport, jsonFactory, googleCredential)
+                .setApplicationName("Google-BucketsInsertExample/1.0").build();
+
+        List<StorageObject> storageObjects = new ArrayList();
+        Storage.Objects.List listObjects = storage.objects().list(bucketName).setPrefix(folderPrefix);
+        Objects objects = listObjects.execute();
+        Assert.assertNotNull(objects.getItems(), "Not found any objects with " + folderPrefix + " prefix.");
```

Comment:
Assertion is blending to others, but breaks the execution. Would be nicer to separate from "real" execution with new lines